correlation_context/HTTP_HEADER_FORMAT.md

```diff
@@ -1,43 +1,71 @@
 # Correlation Context HTTP Header Format
 
-A correlation context header is used to pass the name-value context properties for the trace. This is a companion header for the `traceparent`. The values should be passed along to any child requests. Note that uniqueness of the key within the `Correlation-Context` is not guaranteed. Context received from upstream service may be altered before passing it along.
+The correlation context header is used to propagate user-supplied key-value pairs through a trace.
+A received header MAY be altered to change or add information and it MUST be passed on to all downstream request.
+
+Multiple correlation context headers are allowed. Values can be combined in a single header according to  [RFC 7230](https://tools.ietf.org/html/rfc7230#page-24).
 
 *See [rationale document](HTTP_HEADER_FORMAT_RATIONALE.md) for details of decisions made for this format.*
 
-# Format
 
-## Header name
+# Header Name
+
+Header name: `Correlation-Context`
+
+@TODO: Agree on the header name
+@TODO: Add more information on casing once we agree on the final header name
 
-`Correlation-Context`
 
-Multiple correlation context headers are allowed. Values can be combined in a single header according to the [RFC 7230](https://tools.ietf.org/html/rfc7230#page-24).
+# Header Content
 
-## Header value
+This section uses the Augmented Backus-Naur Form (ABNF) notation of [[!RFC5234]], including the DIGIT rule in <a data-cite='!RFC5234#appendix-B.1'>appendix B.1 for RFC5234</a>. It also includes the `OWS` rule from <a data-cite='!RFC7230#whitespace'>RFC7230 section 3.2.3</a>.
 
-`name1=value1[;properties1],name2=value2[;properties2]`
+## Definition
+
+```
+list        = list-member 0*179( OWS "," OWS list-member )
+list-member = key OWS "=" OWS value *( OWS ";" OWS property )
+property    = key OWS "=" OWS value
+property    = key OWS
+key         = <token, defined in [[RFC2616], Section 2.2](https://tools.ietf.org/html/rfc2616#section-2.2)>
+value       = %x21 / %x23-2B / %x2D-3A / %x3C-5B / %x5D-7E
+              ; US-ASCII characters excluding CTLs,
+              ; whitespace DQUOTE, comma, semicolon,
+              ; and backslash
```

Comment:
What is the rationale for excluding DQUOTE and backslash? Is that taken from set-cookie?

Reply:
Yes that is a direct copy from the RFC https://tools.ietf.org/html/rfc6265#section-4.1

Reply:
Yes, this was for the sake of consistency with `set-cookie`. Also, the backslash is often used for quoting. While this isn't the case for `urlencode`, that might be the reason.